src/mame/includes/cgc7900.h

```diff
@@ -6,12 +6,18 @@
 #define __CGC7900__
```

Comment:
While you're changing this file, could you change the `#include` guard to follow the convention avoiding reserved `__NAMES`?  It should be `MAME_INCLUDES_CGC7900_H`.